lib/rspec/core/configuration.rb

```diff
@@ -787,17 +789,49 @@ def color
         value_for(:color) { @color }
       end
 
+      # Returns the configuration option for forcing color, but should not
+      # be used to check if color is supported.
+      #
+      # @see color_enabled?
+      # @return [Boolean]
+      def force_color
+        value_for(:force_color) { @force_color }
+      end
+
+      # Returns the configuration option for forcing no color, but should not
+      # be used to check if color is supported.
+      #
+      # @see color_enabled?
+      # @return [Boolean]
+      def no_color
+        value_for(:no_color) { @no_color }
+      end
+
       # Check if color is enabled for a particular output.
       # @param output [IO] an output stream to use, defaults to the current
       #        `output_stream`
       # @return [Boolean]
       def color_enabled?(output=output_stream)
-        output_to_tty?(output) && color
+        if force_color
+          true
+        elsif no_color
+          false
+        elsif color
+          tty? || output_to_tty?(output)
+        else
+          output_to_tty?(output)
+        end
       end
 
-      # Toggle output color.
+      # Deprecated: toggle output color.
```

Comment:
That definitely makes sense. The only reference to `tty` I see in this file is the `add_setting :tty` on line 401. That's already marked as `@private` and I don't see it generated in the docs. Thinking I should mark it both `@private` and `@deprecated`? Or make it no longer `@private` so as to say "hey, this exists as you well know, but stop using it`? ;-)